slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/ValidatorsDao.java

```diff
@@ -64,4 +64,15 @@ public boolean isEnabled(final Handle handle, final int validatorId) {
         .mapTo(Boolean.class)
         .first();
   }
+
+  public boolean hasSigned(final Handle handle, final int validatorId) {
+    return handle
+        .createQuery(
+            "SELECT EXISTS(SELECT 1 FROM SIGNED_ATTESTATIONS WHERE validator_id = ?)"
+                + " OR EXISTS(SELECT 1 FROM SIGNED_BLOCKS WHERE validator_id = ?)")
+        .bind(0, validatorId)
```

Comment:
You can simplify this to just bind the one value by using a named binding https://jdbi.org/#_binding_arguments

Reply:
done